### PR TITLE
Case study and team page prototypes

### PR DESCRIPTION
**Pull request by saralohr**
No biggie if these don't get in today, they are just a few prototypes for the case study and team pages.

cc @mrgerardorodriguez @erikjung

### CODE REVIEW

File: src/pages/sandbox/case-studies-core-a.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  </p>
+  <p>
+    We've worked on high-profile projects ranging from Walmart's responsive web redesign to the official Obama '08 iPhone App.
+  <p>
+</div>
+
+{{! Logos }}
+<div class="u-release u-bgGray u-spaceEnds6 u-padEnds3 u-md-padEnds6 u-padSides1">
+  <div class="Grid Grid--alignCenter Grid--alignMiddle u-containSpread">
+    {{#each drizzle.data.case-studies.contents.logos}}
```
Comment (mrgerardorodriguez) [inline src/pages/sandbox/case-studies-core-a.html]: Not sure if this is valid, but could this data be accessed using the `{{data}}` template helper somehow? Here's a wild guess:
```
{{#with (data "case-studies")}}
      {{#each logos}}
         ...
      {{/each}}
{{/with}}
```
I haven't tried that, so I'm curious. :)

**Comment by mrgerardorodriguez** [comment]
Just an inline curiousity question. Otherwise, LGTM! 👍

File: src/pages/sandbox/case-studies-core-a.html (hunk 1 above)
Comment (tylersticka) [inline src/pages/sandbox/case-studies-core-a.html]: I don't know if we're using an up-to-date enough version of Drizzle. 😕 

It's been difficult to prioritize upgrading that stuff, but we probably should at some point.

**Comment by tylersticka** [comment]
I only reviewed the stuff that _isn't_ in sandbox with any depth. LGTM! 👍

File: src/pages/sandbox/case-studies-core-a.html (hunk 1 above)
Comment (mrgerardorodriguez) [inline src/pages/sandbox/case-studies-core-a.html]: > I don't know if we're using an up-to-date enough version of Drizzle.

Oh, right. Good point. 

> It's been difficult to prioritize upgrading that stuff, but we probably should at some point.
 
Maybe? ¯\_(ツ)_/¯

**Comment by saralohr** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;S1EMXmbO&#x2F;174-mockups-prototypes-what-we-ve-done-core"> Mockups&#x2F;Prototypes: What We&#39;ve Done (Core)</a>

**Comment by saralohr** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;1INQ9zmB&#x2F;175-mockups-prototypes-case-study-single"> Mockups&#x2F;Prototypes: Case Study (Single)</a>

**Pull request merged**